bHY6yh-d38035/cod-vyg67d_bHY6yh-d38035.xml

```diff
@@ -0,0 +1,166 @@
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+              <name>Alicia Koepke</name>
+              <resp>Transcription Editor</resp>
+              <resp>TEI Encoder</resp>
+           </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 7</title>
+               <date when="2023-12-14">December 14, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="O" n="cod-vyg67d">Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2021-05-18" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#O" n="85-v"/>
+            <cb ed="#O" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d38035"><!-- p2l5-r7 -->
+            <head xml:id="bHY6yh-d38035-Hd1e108">Rubrica 7</head>
+            <head xml:id="bHY6yh-d38035-Hd1e111" type="question-title">De nocumento susceptionis munerum et beneficiorum</head>
+            <p xml:id="bHY6yh-d38035-d1e114">
+               De nocumento 
+               <lb ed="#O"/>susceptionis munerum et bene<lb ed="#O" break="no"/>ficiorum. Rubrica. 7.
+            </p>
+            <p xml:id="bHY6yh-d38035-d1e121">
+               <lb ed="#O"/><ref xml:id="bHY6yh-d38035-Rd1e117">IN exodo capitulo 23.</ref> <quote xml:id="bHY6yh-d38035-Qd1e125">Munera ex<lb ed="#O" break="no"/>cecant
+               <lb ed="#O"/>etiam prudentes et sub<lb ed="#O" break="no"/>uertunt\
+               <pb ed="#O" n="85-v"/>
+               <cb ed="#O" n="a"/>
+               uerba iustorum.</quote> 
+            </p>
+            <p xml:id="bHY6yh-d38035-d1e135">
+               <lb ed="#V" n="18"/>
```

Comment:
Not sure why we have a line break from V with n="18"; My guess is this is left over from the copied file. So, should we delete it?
```suggestion
```